### PR TITLE
stylelint change rules

### PR DESCRIPTION
**Pull request by serrazina**
Change "block-closing-brace-space-after" to "never"
(Require a single space or disallow whitespace after the closing brace of blocks.)

Change "selector-list-comma-space-after" to "never-single-line" 
(There must never be a single space after the commas in single-line selector lists.)

Removed "selector-no-type" to allow type selectors.

### CODE REVIEW

File: .stylelintrc [hunk 1]
```diff
@@ -84,11 +83,10 @@
     "selector-combinator-space-before": "always",
     "selector-list-comma-newline-after": "always",
     "selector-list-comma-newline-before": "never-multi-line",
-    "selector-list-comma-space-after": "never",
+    "selector-list-comma-space-after": "never-single-line",
     "selector-list-comma-space-before": "never",
     "selector-max-specificity": "0,4,0",
     "selector-no-id": true,
-    "selector-no-type": true,
```
Comment (xipasduarte) [inline .stylelintrc:91]: Not going to happen. This rule, even in the cases where we "must" break it, is a best practice in any good front-end development.

Disable a specific block, dismissed with an argumentative comment, to remove the unwanted warnings.

File: .stylelintrc [hunk 2]
```diff
@@ -84,11 +83,10 @@
     "selector-combinator-space-before": "always",
     "selector-list-comma-newline-after": "always",
     "selector-list-comma-newline-before": "never-multi-line",
-    "selector-list-comma-space-after": "never",
+    "selector-list-comma-space-after": "never-single-line",
```
Comment (xipasduarte) [inline .stylelintrc]: This rule does not make sense. "never" is supposed to play together with `selector-list-comma-newline-after` to produce our required results: one selector per line.

File: .stylelintrc [hunk 3]
```diff
@@ -88,8 +87,8 @@
     "selector-list-comma-space-before": "never",
     "selector-max-specificity": "0,4,0",
     "selector-no-id": true,
-    "selector-no-type": true,
     "selector-no-universal": true,
+    "selector-no-type": true,
```
Comment (xipasduarte) [inline .stylelintrc:90]: It was in the right alphabetical position before 😉

**Comment by xipasduarte** [comment]
I thought we could fall back on `"always-single-line"`  for some of the troublesome rules, but it turns out that there are cases where they force us to act out of place. I removed and adjusted the original rules this pull request handled.

Furthermore, I've corrected some of the genesis-starter's styles based on the warnings.

**Review by s3rgiosan — APPROVED** [review]

**Pull request merged**